website/docs/using-unleash/compliance/soc2.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: SOC2 compliance for feature flags
+description: 'SOC2-compliant feature flags at scale with Unleash.'
+---
+
+# SOC2 compliance
+
+## Overview
+
+To get SOC2 certified and maintain your compliance, you must ensure that any systems you integrate with, including feature flagging solutions, are also SOC2 certified. Using a homegrown or third-party feature flagging system without SOC2 compliance can compromise your certification and introduce unnecessary risks.
+
+This guide provides an overview of how Unleash features align with SOC2 controls, helping your organization meet its compliance requirements.
+ 
+
+## How Unleash features map to SOC2 controls
+
+| SOC2 Control                                | SOC2 Control Description                                                                                             | Unleash Feature                                                                                                                                           |
+|---------------------------------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CC 2.1, CC 7.2 Log management utilized      | The company utilizes a log management tool to identify events that may have a potential impact on the company's ability to achieve its security objectives. | Unleash provides a log of all [events](/reference/events), such as configuration changes and access.                                                                         |
```

Comment:
For the access logs, do we mean [login history](https://docs.getunleash.io/reference/login-history)?

Reply:
Yes, but more generally login history plus event log since access could be via the API.